Makefile

```diff
@@ -383,6 +383,10 @@ docdir:
 .PHONY: docs
 docs: .install.md2man docdir $(MANPAGES) ## Generate documentation
 
+.PHONE: xref_helpmsgs_manpages
+xref_helpmsgs_manpages:
```

Comment:
I'm really unhappy with this. I would like this to depend on `bin/podman`, but since that's a PHONY, adding that dependency causes a rebuild. Is there any clean way to say that this target depends on an existing `bin/podman` but without forcing a recompile?

Reply:
I don't think so, and even if it was possible as an anti-pattern it would likely cause worse problems later on.

It seems like having `bin/podman` as a phony is the real problem...and possibly w/o a good solution that doesn't involve mitigating lots of technical-debt.

This (adding check in CI) might be one of those situations where the best Makefile fix involves plugging your fingers into your ears and singing Marry Poppins :grin: